Core/src/org/sleuthkit/autopsy/datasourceprocessors/xry/AbstractSingleKeyValueParser.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2019 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.datasourceprocessors.xry;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import org.sleuthkit.autopsy.coreutils.Logger;
+import org.sleuthkit.datamodel.BlackboardAttribute;
+import org.sleuthkit.datamodel.Content;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ * Template parse method for reports that make blackboard attributes from a
+ * single key value pair.
+ *
+ * This parse implementation will create 1 artifact per XRY entity.
+ */
+abstract class AbstractSingleKeyValueParser implements XRYFileParser {
+    
+    private static final Logger logger = Logger.getLogger(AbstractSingleKeyValueParser.class.getName());
+
+    private static final char KEY_VALUE_DELIMITER = ':';
+
+    @Override
+    public void parse(XRYFileReader reader, Content parent) throws IOException, TskCoreException {
+        Path reportPath = reader.getReportPath();
+        logger.log(Level.INFO, String.format("INFO: Processing report at [ %s ]", reportPath.toString()));
```

Comment:
Is putting "INFO" in the message intended to make it easier to grep for these messages during field trials? If so, I think that is a good idea, but using something more distinctive would be better.

Reply:
I missed this. During testing all messages were system.out statements. I wanted to add a sense of realism. The intention of these messages is to give context in the log when a severe is encountered.